src/scripts/events/master.js

```diff
@@ -37,7 +36,7 @@ function hasEventEmitter(obj) {
  */
 function detachEvents(target) {
 
-    let hasEvents = target && hasEventEmitter(target) && size(target._events);
+    const hasEvents = target && hasEventEmitter(target) && size(target._events);
```

Comment:
Consider removing the `size` dependency here and updating `size(target._events)` to  `target._eventsCount`.

---

```js
const myEmitter = new MyEmitter();

myEmitter.on('event', function(a, b) {

  console.log(a, b, this);

  // Prints:
  //   a b MyEmitter {
  //     domain: null,
  //     _events: { event: [Function] },
  //     _eventsCount: 1,
  //     _maxListeners: undefined }

});

myEmitter.emit('event', 'a', 'b');

```